vspec2binary.py

```diff
@@ -33,16 +33,29 @@ def usage():
     sys.exit(255)
 
 import os.path
+<<<<<<< HEAD:vspec2binary.py
```

Comment:
You have some conflict markers here that should not be in the commit.
You might just edit this file manually, keep the lines that should be there, and 
```git add vspec2binary.py
git commit --amend
```
to fix the commit.  (Or create a new commit but even with what is there now it would be worthwhile to squash the changes together before merging, IMHO)